samples/graph-activity-feed/csharp/ActivityFeedNotification/Views/Home/Configure.cshtml

```diff
@@ -22,45 +23,47 @@
 </p>
 <script>
 
-    microsoftTeams.initialize();
-    var conextchatId = "";
-    // Check the initial theme user chose and respect it
-    microsoftTeams.getContext(function (context) {
-        conextchatId = context.chatId;
-        console.log(context);
-        if (conextchatId != "") {
-            document.getElementById("textConfiguration").innerHTML = "Configure your Group Chat Tab to Send Activity Feed Notification: ";
-             $("#teamsTab").hide();
-             $("#groupChatTab").show();
-        }
-        else {
-            document.getElementById("textConfiguration").innerHTML = "Configure your Teams Tab to Send Activity Feed Notification: ";
-             $("#groupChatTab").hide();
-             $("#teamsTab").show();
-        }
-        if (context && context.theme) {
-            setTheme(context.theme);
-        }
-    });
+    microsoftTeams.app.initialize().then(() => {
+        var conextchatId = "";
+        // Check the initial theme user chose and respect it
+        microsoftTeams.getContext(function (context) {
```

Comment:
shouldn't this be `microsoftTeams.app.getContext`?

Reply:
Were you able to verify this worked as expected?

Reply:
changes this to v2 implementation